netlify-cms@2/packages/netlify-cms-core/src/reducers/entryDraft.js

```diff
@@ -117,7 +118,29 @@ function entryDraftReducer(state = Map(), action) {
         state.set(
           'hasChanged',
           !entries.some(e => newData.equals(e.get(...dataPath))) ||
-            !entries.some(e => newMeta.equals(e.get('meta'))),
+          !entries.some(e => newMeta.equals(e.get('meta'))),
+        );
+      });
+    }
+    case DRAFT_FOCUS_FIELD: {
+      return state.withMutations(state => {
+
+        const { field, value, metadata, entries, i18n } = action.payload;
+        const name = field.get('name');
+        window.previewStyle = {
```

Comment:
Sendo uma variável global precisamos de um nome melhor (scoped e mais descritivo), talvez:

```suggestion
        window.storefrontCmsFocusField = {
```

Reply:
Alterado